engine/classes/Elgg/UpgradeService.php

```diff
@@ -144,9 +176,6 @@ protected function upgradeCode($version, $quiet = false) {
 	 * @return mixed
 	 */
 	protected static function includeCode($file) {
-		// do not remove - some upgrade scripts depend on this
-		global $CONFIG;
```

Comment:
It was added in 1.x branch as workaround for #7457. Later the underlying issue #7613 was also fixed. So I believe it is not needed anymore.